packages/dapp/src/components/utility/FormElements.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import styled from "styled-components";
+
+export const StyledFormContainer = styled.div`
+  margin: 1em 0;
```

Comment:
why ems?

Reply:
No particular reason, I just wanted to give the component a little room to breathen so I added something quick w/o real thought behind it. 

I figure we'll be updating these when we add actual styles/design and can use `rem` or `px` at that point.